### DISCUSSION TITLE
Is there a "one window mode"? Or: How to avoid Ray creates new windows?

### DISCUSSION THREAD

**Discussion by ckilb** [spatie/ray#851] [Q&A]
I have the issue that Ray sometimes (I couldn't figure out when it exactly happens) creates a new window instead of reusing the already existing window.

This is confusing because normally I don't recognize a new window has been opened and wonder if there's is a bug in my code because the dumps I expect to be recorded don't appear in the old window.

Why does Ray some times create new windows?
Is there a way to avoid this?

**Comment by freekmurze** [comment]
In the settings, you can choose the let Ray never open up a new window 👍

![CleanShot 2023-10-10 at 15 11 32@2x](https://github.com/spatie/ray/assets/483853/e3508a46-942e-4b10-8412-2e029503754b)
